src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -42,46 +43,60 @@ def _bearer_auth_str(token):
     return 'Bearer ' + token
 
 
-def _headers(username, password):
+def _headers(username, password, is_manifest=False):
     if username is None:
         auth = _bearer_auth_str(password)
     else:
         auth = _basic_auth_str(username, password)
 
+    if is_manifest:
+        return {'Authorization': auth,
+                'Accept': DOCKER_MANIFEST_V2}
     return {'Authorization': auth}
 
 
-def _delete_data_from_registry(login_server,
-                               path,
-                               username,
-                               password,
-                               resultIndex=None,  # pylint: disable=unused-argument
-                               retry_times=3,  # pylint: disable=unused-argument
-                               retry_interval=5):  # pylint: disable=unused-argument
-    registryEndpoint = 'https://' + login_server
+def _params(count=20):
```

Comment:
This function name is too generic and will lead to ambiguity. Please make it more accurate.